src/autopas/containers/verletClusterCells/VerletClusterCells.h

```diff
@@ -395,7 +395,8 @@ class VerletClusterCells : public ParticleContainer<FullParticleCell<Particle>>,
     std::vector<Particle> invalidParticles;
 
     for (size_t i = 0; i < this->_cells.size(); ++i) {
-      for (auto &p : this->_cells[i]._particles) {
+      for(auto it = this->_cells[i].begin(); it != this->_cells[i].end(); ++it) {
+        Particle &p = *it;
```

Comment:
also here why change the way particles are accessed?

Reply:
because the particles are accessed directly and not through the interface. the reference container contains references though so you can't access them via the dot notation.